src/tests/enforce-boolean-naming-prefixes.test.ts

```
@@ -0,0 +1,451 @@
+      errors: [
+        {
+          messageId: 'missingBooleanPrefix',
+          data: {
+            type: 'function',
+            name: 'valid',
+            prefixes: 'is, has',
+          },
+        },
+      ],
+    },
+  ],
```

ESLint

[error] Replace `··` with `····`

(prettier/prettier)